docker-compose.dev.yaml

```diff
@@ -0,0 +1,8 @@
+version: '3.8'
+
+services:
+  mongodb:
+    ports:
+      - "27017:27017"
+    volumes:
+      - ~/apps/moviecenter/mongo:/data/db
```

Comment:
Why would you want your data to persist in dev? That would only leave you open for bugs due to dirty state

Reply:
It was convenient when needed to check that everything works after a build.

I'm running the tests with `docker-compose.test.yaml` that does not persist the data.